tests/models/test_tpu.py

```diff
@@ -495,3 +495,22 @@ def teardown(self, stage):
 
     model = DebugModel()
     tpipes.run_model_test(trainer_options, model, on_gpu=False, with_hpc=False)
+
+
+@RunIf(tpu=True)
+@pl_multi_process_test
+def test_predict_tpu_multi_cores(tmpdir):
```

Comment:
this test makes no assertions, which means it will pass if the trainer does not train or the prediction loop does not return any predictions

Reply:
Added an assertion for trainer state.